tests/test_views.py

```diff
@@ -1,21 +1,87 @@
 """
-Tests for the frontend.
+Unit tests for the frontend code.
 """
 
 import unittest
 
-import ga4gh.server as server
+import ga4gh.backends
 import ga4gh.cli
+import ga4gh.protocol as protocol
+import ga4gh.server as server
+
+
+class MockBackend(ga4gh.backends.Backend):
+    """
+    A mock Backend class for testing.
+    """
+    def __init__(self, dataDir):
+        self._dataDir = dataDir
+
+    def searchVariants(self, request):
+        response = protocol.GASearchVariantsResponse()
+        return response
+
+    def searchVariantSets(self, request):
+        response = protocol.GASearchVariantSetsResponse()
+        return response
+
 
 class TestFrontend(unittest.TestCase):
 
     def setUp(self):
+        server.app.config['TESTING'] = True
+        server.app.config['VariantBackend'] = MockBackend('/tmp/')
```

Comment:
I think it would be a bit better if we had `MockBackend()` here instead of providing /tmp as a dummy parameter. I think you can do this by just `def __init__(self, dataDir=None):` for the constructor above.